src/views/payment-sheet-views/card-view.js

```diff
@@ -114,6 +113,33 @@ CardView.prototype._sendRequestableEvent = function () {
   }
 };
 
+CardView.prototype._renderCardIcons = function () {
+  var overrides = this.merchantConfiguration.overrides;
+  var cardIcons = this.getElementById('card-view-icons');
+  var supportCardBrands;
+
+  cardIcons.innerHTML = cardIconHTML;
+
+  try {
+    supportCardBrands = overrides.fields.number.supportedCardBrands;
+
+    if (supportCardBrands) {
+      Object.keys(supportCardBrands).forEach(function (cardBrand) {
+        var value = supportCardBrands[cardBrand];
+        var selector, iconDiv;
+
+        if (value === false) {
+          selector = 'div[data-braintree-id="' + constants.cardTypeIcons[cardBrand] + '-card-icon"]';
+          iconDiv = document.querySelector(selector);
+          hideCardIcon(iconDiv);
+        }
+      });
+    }
+  } catch (error) {
```

Comment:
do we think it's worth sending an analytic log if we ever hit this error block?

Reply:
Talked through the implementation. Since many merchants don't set overrides, we really should be swallowing this error. And from a readability standpoint, this reads better than checking if there's overrides, _and_ fields, _and_ number, etc etc.

We'll add a comment to add some context on why we're swallowing errors in the try/catch block and should be good to go from there.